nilearn/decoding/fista.py

```diff
@@ -231,7 +236,7 @@ def mfista(f1_grad, f2_prox, total_energy, lipschitz_constant, w_size,
             #
             # For this reason, we rely more on the linesearch-like
             # strategy to set the dgap_tol
-            dgap_tol = abs(energy_delta) / (i + 1.)
+            dgap_tol = abs(energy_delta)
```

Comment:
This looks really violent. At this point, we are 100% heuristic, and what is left of the convergence theory disappears. Will be nice to see convergence curves.